build/circle-test.sh

```diff
@@ -49,14 +51,14 @@ fi
 # better.
 if [ "${CIRCLE_TEST_REPORTS}" != "" ]; then
     if [ -f "${outdir}/test.log" ]; then
-	mkdir -p "${CIRCLE_TEST_REPORTS}/go"
-	${builder} go2xunit < "${outdir}/test.log" \
-	       > "${CIRCLE_TEST_REPORTS}/go/test.xml"
+    mkdir -p "${CIRCLE_TEST_REPORTS}/go"
```

Comment:
what is an 8 space tab character? a tab is a tab, no? Anyway, I've replaced all the spaces with tabs.